### DISCUSSION TITLE
Log File Location

### DISCUSSION THREAD

**Discussion by ghost** [Cinnamon/kotaemon#398] [Q&A]
A quick one. Where are the log files located on a Mac? thx for any help.

**Comment by taprosoft** [comment]
All data are stored in `ktem_app_data` folder in you root `kotaemon` install directory. Currently logs file are not enabled explicitly.

**Reply by ghost** [reply]
I just finished 4 hours of indexing a graph db and got this.
└── create_final_community_reports❌ Errors occurred during the pipeline run, see logs for more details.

So logging is not enabled? How do I do that? can't find it in the docs.

**Reply by ghost** [reply]
Anybody??

**Comment by drodel** [comment]
Hello @motigib,

Are you still having this issue?

I am running into a similar issue.
└── create_base_entity_graph❌ Errors occurred during the pipeline run, see logs for more details.

The output from the docker container says the below. I am wondering if the warning is triggering the message.
    Initializing project at /app/ktem_app_data/user_data/files/graphrag/6a0f3fda-2ae0-4acf-8519-99366ad7a2d8

    /usr/local/lib/python3.10/site-packages/numpy/core/fromnumeric.py:59: FutureWarning: 'DataFrame.swapaxes' is deprecated and will be removed in a future version. Please use 'DataFrame.transpose' instead.
      return bound(*args, **kwds)

I have found this page related to settings, but have not figured out how to enable the logging.

https://cinnamon.github.io/kotaemon/pages/app/settings/overview/

I found "flowsettings.py" in the root of the repo, but nothing caught my eye as enabling logging.

**Reply by ghost** [reply]
I still have the problem and no log settings to be found.

**Reply by points-unknown** [reply]
graphrag logs are stored under "user-data", which may or may not be in your root folder, depending on your setup. (it's defined in 'flowsettings.py'.  
,,,   /user-data/files/graphrag/

then each uuid under there has the logs from that model run.
